BTrees/tests/common.py

```diff
@@ -77,6 +94,38 @@ def _closeRoot(self, root):
         transaction.abort()
         root._p_jar.close()
 
+    def testPersistentSubclass(self):
+        # Can we subclass this and Persistent?
+        # https://github.com/zopefoundation/BTrees/issues/78
+        import persistent
+
+        class PersistentSubclass(persistent.Persistent):
+            pass
+
+        __traceback_info__ = self._getTargetClass(), persistent.Persistent
+        type('Subclass', (self._getTargetClass(), PersistentSubclass), {})
+
+    def testPurePython(self):
+        import importlib
+        kind = self._getTargetClass()
+        class_name = kind.__name__
+        module_name = kind.__module__
+        module = importlib.import_module(module_name)
```

Comment:
Presumably if you already have a class from the module, then the module itself is already in `sys.modules[module_name]`.